### DISCUSSION TITLE
H4 v7 Problem with closed circuit motor that stops after little use

### DISCUSSION THREAD

**Discussion by tektonandroid** [kachurovskiy/nanoels#173] [Q&A]
Good morning.  I just installed the H4 on my small lathe.  WORKS!!!.
 But I have a problem: The Z axis, even without the carriage attached, consumes 117mA at rest.  When activated, it goes up to 0.5 mA in void.  So far, everything is normal, but after having it spin for a while (1 minute, or even less, since it is quite random) the motor stops responding and consuming, leaving it to rotate freely.  It is not the power supply, which continues to deliver 48V.  If I turn off the power and reconnect it, it works again... until now, it has not recovered and the engine alarm remains intermittent.
 In the assembly instructions, the motor display is connected to +, and the encoder display to -.  Correct?
 Does anyone know what I can check?
 The engine characteristics are these:
![-1088237962607083225](https://github.com/kachurovskiy/nanoels/assets/133983512/e680c878-6621-45e7-98b0-21c3c5a358b4)

https://github.com/kachurovskiy/nanoels/assets/133983512/9e313a99-1950-4136-985c-e870fb8d2224

Saludos.

**Comment by tektonandroid** [comment]
From what I have been seeing in the instructions, it is an exceeded current alarm...But I don't understand why, since I have not exceeded 1.4A (with the half nut attached)
![-10896254771501104946](https://github.com/kachurovskiy/nanoels/assets/133983512/94bce0f1-417e-4f34-ac3f-b37e8fdbd090)
I'm a total newbie to this engine stuff, so I'd appreciate any help.
 Thank you all.

**Comment by kachurovskiy** [comment]
Hi @tektonandroid, 

- Check if there's a mechanical problem somewhere that overloads the motor
- Check for any loose wires / connections - this is often the case
- Try contacting your seller and checking if they're amendable to a refund / replacement
- How are you measuring the current when motor is running? A current clamp over 1 (not 2) of the supply wires seems appropriate here
- What's the current your power supply is rated for?

**Reply by tektonandroid** [reply]
Hello, Maxim. Motor was working without charge (half nut open).
 Wires of the power supply are  in good conditions. Wires to control has a poor instalation, at the moment, but are no shortcircuits in there. 
I measured current with a tester with max and min register, only in 1 wire, and the power supply is adjusted to 48,7 v (this is the only factor i suspect guilty because motor is 48v max) and 5 Amps.
Motor alarm cleared  for 5 or 10 times, but now, alarm don't clear anymore...
Thanks for your answer.
Teo.

**Reply by tektonandroid** [reply]
Finally, I have received a new engine.  I have lowered the voltage to 38V and tested it without mounting.  No problem.  Then I assembled it and turned the screw without the carriage half nut attached and it turned without problems.  But when I attached the nut the motor stopped, giving an overcurrent alarm.  After uncoupling it, it did it again and the third time I uncoupled the half wheel, it no longer gave me any problems... The truth is that the carriage does not have strange bearings or hardness that would prevent the spindle from turning, so it doesn't I know it could have happened.  Now I have to deal with the h4 settings, because it doesn't do what I want...I'm closing this matter.  Thank you all.
